js/components/ViewPagerIndicator.js

```diff
@@ -0,0 +1,84 @@
+import React, { PropTypes } from 'react';
+import {
+  View,
+  Animated,
+  Dimensions,
+  StyleSheet,
+} from 'react-native';
+
+const { width } = Dimensions.get('window');
+const DOT_SIZE = 12;
```

Comment:
These are only UX specs, not visual, so don't worry about implementing them to pixel perfection. Android page indicators look half the size to me.